src/main/java/bowling/BowlingApplication.java

```diff
@@ -0,0 +1,40 @@
+package bowling;
+
+import bowling.domain.BowlingGame;
+import bowling.domain.frame.Frames;
+import bowling.domain.player.Player;
+import bowling.ui.InputView;
+import bowling.ui.ResultView;
+
+public class BowlingApplication {
+
+    public static void main(String[] args) {
+
+        String player = InputView.inputPlayerName();
+        BowlingGame bowlingGame = new BowlingGame(new Player(player), new Frames());
```

Comment:
```suggestion
        BowlingGame bowlingGame = new BowlingGame(player);
```

BowlingGame 클래스를 사용하는 클라이언트 입장에서 Player Frames에 대해 알고 있어야지 BowlingGame을 만들 수 있겠네요!
해당 클래스를 사용하는 클라이언트 입장에서 몰라도 되는 정보가 있다면, 추상화해볼 수 있을 것 같습니다 ㅎㅎ

Reply:
BowlingGame 내부에서 충분히 처리할 수 있는 일을 외부에서 하도록 하고 있었네요!
피드백 주신대로 수정해보겠습니다.